dev/com.ibm.ws.jsonb_fat/fat/src/com/ibm/ws/jsonb/fat/JSONPContainerTest.java

```diff
@@ -49,6 +52,7 @@ public static void tearDown() throws Exception {
     }
 
     @Test
+    @SkipForRepeat(EE9_FEATURES)
```

Comment:
Can you add a comment indicating why this test is being skipped for EE9 features?

Reply:
done, thanks!